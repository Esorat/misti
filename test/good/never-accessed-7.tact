fun test1(): Int {
    let a: Int = 20; // never-accessed variable
    let b: Int = 22; // read-only variable
    return b;
}
