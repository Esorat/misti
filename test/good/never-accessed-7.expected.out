test/good/never-accessed-7.tact:2:5:
  1 | fun test1(): Int {
> 2 |     let a: Int = 20; // never-accessed variable
          ^
  3 |     let b: Int = 22; // read-only variable
Variable is never accessed
Help: Consider removing the variable
See: https://nowarp.io/tools/misti/docs/detectors/NeverAccessedVariables